runtime/runtime-params-estimator/src/utils.rs

```diff
@@ -19,6 +21,14 @@ pub fn read_resource(path: &str) -> Vec<u8> {
         .unwrap_or_else(|err| panic!("failed to load test resource: {}, {}", path.display(), err))
 }
 
+/// Attempts to clear OS page cache on Linux based system. Will fail on
+/// other systems. Requires write access to /proc/sys/vm/drop_caches
+pub fn clear_linux_page_cache() -> io::Result<()> {
+    let out = Command::new("sync").output().expect("sync failed");
```

Comment:
To the best of my knowledge this is not necessary. That said, if you'd like to keep it still, I'd change this to call `libc::sync` instead, which is among other things is infallible.

Reply:
Hm, I was not able to find anything confirming tor deny the necessity of a sync here. From [man proc(5)](https://man7.org/linux/man-pages/man5/proc.5.html)

>  /proc/sys/vm/drop_caches (since Linux 2.6.16)
              Writing to this file causes the kernel to drop clean
              caches, dentries, and inodes from memory, causing that
              memory to become free.  This can be useful for memory
              management testing and performing reproducible filesystem
              benchmarks.  Because writing to this file causes the
              benefits of caching to be lost, it can degrade overall
              system performance.

It sounds to me that recently written pages, that might still be dirty, would not necessarily be cleaned. So to be sure, I would like to keep the sync.

Using libc makes a lot of sense, though! I also decided to put it behind some `cfg!`, seems cleaner.